src/react-components/info-dialog.js

```diff
@@ -27,6 +27,21 @@ class InfoDialog extends Component {
 
     const loc = document.location;
     this.shareLink = `${loc.protocol}//${loc.host}${loc.pathname}`;
+    this.onKeyPress = this.onKeyPress.bind(this);
+  }
+
+  componentDidMount() {
+    window.addEventListener("keypress", this.onKeyPress);
```

Comment:
can you change this to `keydown`? it doesn't work in Chrome (tested on macOS).

the `keypress` event does fire correctly in both Firefox and Safari. in Chrome, it turns out that the `keypress` event is emitted for character keys, but evidently not for such keys as `Escape`, `ArrowDown`, etc.